From 2e0fc8c850e2fe569a130c21ba5a78619b324682 Mon Sep 17 00:00:00 2001
From: "Mishra, ManishX" <manishx.mishra@intel.com>
Date: Thu, 21 Oct 2021 14:23:12 +0530
Subject: [PATCH] Revert "Adding a vendor property to extract host_ip address
 vendor.intel.ipaddr"

This reverts commit 729cf92971b0a604920ac4ffa6869dd52446f0eb.
---
 aafd/logwrapper.te                  | 1 -
 aafd/vendor_init.te                 | 1 -
 sensors/mediation/property.te       | 1 -
 sensors/mediation/property_contexts | 1 -
 4 files changed, 4 deletions(-)
 delete mode 100644 sensors/mediation/property.te
 delete mode 100644 sensors/mediation/property_contexts

diff --git a/aafd/logwrapper.te b/aafd/logwrapper.te
index 07ce02f..df7da1b 100644
--- a/aafd/logwrapper.te
+++ b/aafd/logwrapper.te
@@ -22,7 +22,6 @@ set_prop(logwrapper, vendor_fixed_perf_prop)
 allow logwrapper p9fs:file r_file_perms;
 allow logwrapper p9fs:dir r_dir_perms;
 set_prop(logwrapper, vendor_suspend_prop)
-set_prop(logwrapper, vendor_intel_ipaddr_prop)
 set_prop(logwrapper, vendor_graphics_gles_prop)
 set_prop(logwrapper, vendor_media_target_prop)
 set_prop(logwrapper, vendor_display_prop)
diff --git a/aafd/vendor_init.te b/aafd/vendor_init.te
index 74f7521..0db94e3 100644
--- a/aafd/vendor_init.te
+++ b/aafd/vendor_init.te
@@ -1,6 +1,5 @@
 set_prop(vendor_init, vendor_video_codec_prop)
 set_prop(vendor_init, vendor_suspend_prop)
-set_prop(vendor_init, vendor_intel_ipaddr_prop)
 get_prop(vendor_init, vendor_hwcomposer_prop)
 set_prop(vendor_init, vendor_usb_controller_prop)
 get_prop(vendor_init, vendor_fixed_perf_prop)
diff --git a/sensors/mediation/property.te b/sensors/mediation/property.te
deleted file mode 100644
index 1a9cdb5..0000000
--- a/sensors/mediation/property.te
+++ /dev/null
@@ -1 +0,0 @@
-vendor_internal_prop(vendor_intel_ipaddr_prop)
diff --git a/sensors/mediation/property_contexts b/sensors/mediation/property_contexts
deleted file mode 100644
index 08e6ee4..0000000
--- a/sensors/mediation/property_contexts
+++ /dev/null
@@ -1 +0,0 @@
-vendor.intel.ipaddr u:object_r:vendor_intel_ipaddr_prop:s0
-- 
2.33.1

